node/src/components/linear_chain_sync/traits.rs

```diff
@@ -0,0 +1,37 @@
+use crate::{
```

Comment:
I think the entire `linear_chain_sync` component has been removed on the `feat-fast-sync` branch.